lifelines/tests/test_estimation.py

```diff
@@ -4841,3 +4842,43 @@ def test_ci_calculation_against_sas(self, fitter, duration, event_observed):
 
         fitter.fit(duration, event_observed, event_of_interest=2)
         npt.assert_allclose(ci_from_sas, np.array(fitter.confidence_interval_))
+
+
+class CureModelExponential(ParametricUnivariateFitter):
+    _fitted_parameter_names = ["cured_", "lambda_", "rho_"]
+
+    def _cumulative_hazard(self, params, times):
+        cured_, lambda_, rho_ = params
+        c = expit(cured_)
+
+        sf = anp.exp(-((times / lambda_) ** rho_))
+        return -anp.log(c + (1 - c) * sf)
+
+
+class TestMixtureCureFitter:
+    def test_exponential_data_produces_correct_inference_for_both_cure_and_non_cure_fractions(self):
+        N = 1000000
+        scale = 5
+        T = np.random.exponential(scale, size=N)
+        observed = np.ones(N, dtype=bool)
+
+        # Censor the data at time = 6
```

Comment:
8?

Reply:
This is a great example of why tests are better documentation than comments. Harder to get out of date. In this case... instantly. Anyway, fix pushed.